src/main/java/com/ericsson/ei/controller/AuthControllerImpl.java

```diff
@@ -52,7 +54,7 @@
         } catch (Exception e) {
             String errorMessage = "Failed to check if security is enabled. Error message:\n" + e.getMessage();
             LOGGER.error(errorMessage, e);
-            return new ResponseEntity<>(errorMessage, HttpStatus.INTERNAL_SERVER_ERROR);
+            return new ResponseEntity<>(ResponseMessage.createJsonMessage(errorMessage), HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
The Logger.error should include the stacktrace as well i think. Same for all other places :)

The message back to user user should not include the e.getMessage, i think it only should be a "handwritten errormessage" since it is an internal server error.  For example, "Internal server error. Failed to check if security is enabled.  Please report the issue or try again later."

Reply:
stacktrace is included in logging by adding the ,e. I think

Reply:
Done.

Reply:
Something else is returned with e, I don't remember what, maybe the error type. I use an apache method to get the trace stringified.